### PR TITLE
Docs: Add desc

### LINKED ISSUE #6: Docs: Add desc
## 개요
Github Pull-Request 협업

## 세부 내용
3. 프로젝트 설명(구현한 기능 및 사진 등)

### PR DESCRIPTION
**Pull request by seungjun222**
close #6

### CODE REVIEW

**Comment by seungjun222** [comment]
굿

**Comment by JitHoon** [comment]
😎

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 
 ## 3. 프로젝트 설명 (구현한 기능 및 사진 등)
-- 
+- - 깃허브 로그인, 정보 공유 게시판, 질문 게시판 등
```
Comment (LikeFireAndSky) [inline README.md:12]: 확인했습니다!!

**Comment by lilviolie** [comment]
🙃

**Review by LikeFireAndSky — COMMENTED** [review]
👍👍

**Pull request merged**